test/fixtures/templater/jinja_exclude_macro_path/macros/macros_exclude/macro_exclude_2.sql

```diff
@@ -0,0 +1,3 @@
+{% materialization my_materialization_name_2, default %}
+ -- materialization...
+{% endmaterialization %}
```

Comment:
Rather than a second `materialization` example here, could you change the second file in this folder to be something which overrides `foo1` and `foo2`, for example something like:

```jinja
{%- macro foo1() -%}105{%- endmacro -%}
{%- macro foo2() -%}106{%- endmacro -%}
```

...to further demonstrate that this folder is not overriding the others?